### DISCUSSION TITLE
how to use <style lang='scss'> in vue sfc？

### DISCUSSION THREAD

**Discussion by fgcp020** [FranckFreiburger/vue3-sfc-loader#47] [Q&A] [answered]
i want to use scss style in  vue SFC file, but how to do that use loadModule?

**Comment by FranckFreiburger** [comment]
Hi fgcp020,

Maybe you can use the following example https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/examples.md#using-another-style-language-stylus and replace stylus with https://github.com/medialize/sass.js/

**Comment by fgcp020** [comment]
Thank you for your reply!

I've tried the sass.js, but it seems it's API does not support ... Woul you please try it ?

sincerely.

**Comment by FranckFreiburger** [comment, marked as answer]
At the moment I have no decent solution for this.
I cannot find a JS sass implementation with a sync API. The closest solution is https://github.com/sass/dart-sass/issues/25

**Reply by FranckFreiburger** [reply]
fell free to subscribe to https://github.com/FranckFreiburger/vue3-sfc-loader/issues/54

**Reply by fgcp020** [reply]
Thank you for your replay!

**Reply by whitedreams47** [reply]
hello. are there any news on this feature?
